handbook/engineering/security/index.md

```diff
@@ -89,11 +89,31 @@ On the security team, we work by planning, tracking, and reviewing - creating a
    
 ### Learning
 
-After the each release, we hold a [retrospective](https://about.sourcegraph.com/retrospectives). We try to understand the degree to which we achieved the goals we communicaed at the beginning of the iteration.  We idenitfy what went well and what our opportunities for imporvement.  We actively choose one of the things we've learned, and target its improvement.
+After the each release, we hold a [retrospective](https://about.sourcegraph.com/retrospectives). We try to understand the degree to which we achieved the goals we communicated at the beginning of the iteration.  We identify what went well and what our opportunities for improvement.  We actively choose one of the things we've learned, and target its improvement.
+
+### Working with other teams
+
+The security team helps other teams investigate, orient, review, and sometimes plan changes. 
+
+**Small changes**
+
+Small changes, defined as less than 1 day of effort are communicated in a GitHub issue, either in the [sourcegraph repository](https://www.github.com/sourcegraph/sourcegraph) or, when sensitive in nature, the [security repository](https://www.github.com/sourcegraph/security-issues).  We provide a detailed bug report, including the steps to reproduce, and an example of the desired state.  Where possible, we identify and propose a fix, and submit a pull request to the [code owner](../code_reviews.md#who-should-i-get-a-code-review-from).
+
+**Medium changes**
+
+Medium changes operate on the same principle as small changes, with the exception of the team not submitting code changes. These changes also include a documented test plan for validation of the security change. 
+
+**Large changes**
+
+Large changes are communicated through [RFCs](../../communication/rfcs/index.md).  The RFC provide as much context as possible about the issue, the reason for the change, and its relative importance and urgency.  We collaborate on the RFC as a team, sharing the RFC with the Engineering Manager(s) for the impacted team for both feedback and [prioritizing](../../product/prioritizing.md).  Upon approval, we create GitHub issues, labeling them with a new label, for the RFC (i.e rfc214), sharing with the Engineering manager.  When possible, a member of the security team attends the planning session, to help provide input should more be needed.
+
+**Providing input**
+
+When input is needed from the team on a change the team is added to a pull request.  We also respond to RFCs when assigned.
```

Comment:
```suggestion
Teams planning and executing changes should ask for Pull Request reviews from the Security Team as needed to gain confidence in the implementation. Teams should also feel free to ask for a security review of any RFC they're working on that aren't directly created from the Security Goals.
```

Reply:
I like the active voice change you've highlighted in your writing. It feels like a higher degree of ownership - thank you for sharing that.